tools/interactive/interactivetool_napari.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="interactive_tool_napari" tool_type="interactive" name="napari" version="@VERSION@+galaxy0" profile="23.1">
+	<description>cell image analysis software</description>
+    <macros>
+        <token name="@VERSION@">0.4.18</token>
+    </macros>
+    <requirements>
+	    <container type="docker">quay.io/sunyi000/napari-galaxy:v0.4.18</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="Napari" requires_domain="True">
+            <port>5800</port>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code">
+    <![CDATA[
+        export HOME=\$PWD &&
```

Comment:
I think this is not needed with a recent profile version (which you have included). 
Can you test this without this line or should we proceed and remove it later?

Reply:
i think this is needed. If i comment it out, I don't see the input images..